test/Microsoft.AspNetCore.Mvc.Core.Test/RequireHttpsAttributeTests.cs

```diff
@@ -45,12 +44,8 @@ public static IEnumerable<object[]> RedirectToHttpEndpointTestData
                 data.Add("localhost", "/pathbase", "/path", "?foo=bar", "https://localhost/pathbase/path?foo=bar");
 
                 // Encode some special characters on the url.
-                // Two paths hit aspnet/External#50 with Mono on Mac.
-                if (!TestPlatformHelper.IsMac || !TestPlatformHelper.IsMono)
-                {
-                    data.Add("localhost", "/path?base", null, null, "https://localhost/path%3Fbase");
-                    data.Add("localhost", null, "/pa?th", null, "https://localhost/pa%3Fth");
-                }
+                data.Add("localhost", "/path?base", null, null, "https://localhost/path%3Fbase");
```

Comment:
No longer need to do `data.Add` anymore. Can change this entire set to be like the other data properties we have:

`return new TheoryData....`